apps/spruce/src/pages/image/ImageEventLog/ImageEventLog.tsx

```diff
@@ -0,0 +1,68 @@
+import styled from "@emotion/styled";
+import Card from "@leafygreen-ui/card";
+import { Subtitle } from "@leafygreen-ui/typography";
+import { LoadingButton } from "components/Buttons";
+import { size } from "constants/tokens";
+import { ImageEvent } from "gql/generated/types";
+import { Header } from "./Header";
+import { ImageEventLogTable } from "./ImageEventLogTable";
+
+type ImageEventLogProps = {
+  allEventsFetched: boolean;
+  events: ImageEvent[];
+  handleFetchMore: () => void;
+  loading?: boolean;
+};
+
+export const ImageEventLog: React.FC<ImageEventLogProps> = ({
+  allEventsFetched,
+  events,
+  handleFetchMore,
+  loading,
+}) => {
+  const allEventsFetchedCopy =
+    events.length > 0 ? "No more events to show." : "No events to show.";
+
+  return (
+    <Container>
+      {events.map((event) => {
+        const { amiAfter, amiBefore, entries, timestamp } = event;
+        return (
+          <ImageEventLogCard
+            key={`event_log_${timestamp}`}
+            data-cy="image-event-log-card"
+          >
+            <Header
+              amiAfter={amiAfter}
+              amiBefore={amiBefore ?? ""}
+              timestamp={timestamp}
+            />
+            <ImageEventLogTable entries={entries} />
+          </ImageEventLogCard>
+        );
+      })}
+      {!allEventsFetched && events.length > 0 && (
+        <LoadingButton
+          data-cy="load-more-button"
+          loading={loading}
+          onClick={handleFetchMore}
+          variant="primary"
+        >
+          Load more events
+        </LoadingButton>
+      )}
+      {allEventsFetched && <Subtitle>{allEventsFetchedCopy}</Subtitle>}
```

Comment:
It looks like the text got off centered:

![Screenshot 2024-08-20 at 11 35 00 AM](https://github.com/user-attachments/assets/d2cba560-0a09-4d95-87f9-15e41e7a287f)